### ISSUE TITLE
Acute respiratory distress syndrome

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6751]
**Preferred term label:**
Acute respiratory distress syndrome

**Synonyms**

**Definition (free text, please give PubMed ID)**
-- as a feature of other diseases such as COVID

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7852066/
19/82

also Acute coronary syndrome as a feature, 4/82

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
